go/interpreter/lfvm/memory.go

```diff
@@ -222,22 +121,57 @@ func (m *Memory) CopyData(offset uint64, trg []byte) {
 	}
 }
 
-func (m *Memory) GetSlice(offset, size uint64) []byte {
+// getSlice obtains a slice of size bytes from the memory at the given offset.
+// Expands memory as needed and charges for it.
```

Comment:
This [comment](https://github.com/Fantom-foundation/Tosca/pull/761#discussion_r1766363076) got lost.